src/Tizen.NUI.Components/Controls/SelectButton.cs

```diff
@@ -208,16 +208,16 @@ public FloatSelector CheckImageOpacitySelector
         /// <since_tizen> 6 </since_tizen>
         /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public Size2D CheckImageSize2D
+        public Size CheckImageSize
```

Comment:
What does the "Check" prefix mean?  Should update in the comments what the CheckImage is; an icon, background, selected or unselected image.
Should this have been renamed when the button got renamed to a SelectButton?

Reply:
Checking at previous emails did we not want this to be the "icon", looks like the "Check" was not updated in all files when the button was renamed to SelectButton